app/mailers/application_mailer.rb

```diff
@@ -7,4 +11,36 @@ class ApplicationMailer < ActionMailer::Base
                             )}>"
                           }
   layout "mailer"
+
+  private
+
+    # Configures the template_path for mails such that email views in our
+    # custom `app/frontend/` folder structure can be found.
+    #
+    # Example for `feedback_mailer.rb`:
+    #
+    # - The default view path is `app/views/`. Inside this folder, the default
+    #   Rails convention is to look for a folder with the class name, that is
+    #   in our case: "feedback_mailer". That is, we obtain a path like
+    #   `app/views/feedback_mailer/`. Inside this folder, Rails will
+    #   look for a file that starts with the action name,
+    #   e.g. "new_user_feedback_email".
+    #
+    # - In addition, we've added `app/frontend/` to the view paths. But here,
+    #   we DON'T want to have a structure like
+    #   `app/frontend/feedback_mailer/new_user_feedback_email.text.erb`
+    #   next to our `app/frontend/feedbacks/` folder.
+    #
+    #   Instead, we modify the template path to just read "feedbacks"
+    #   (instead of "feedback_mailer"), therefore obtaining the correct path:
+    #   `app/frontend/feedbacks/new_user_feedback_email.text.erb`.
+    def mail(headers = {}, &)
+      usual_rails_template_path = self.class.name.underscore
+      custom_template_path = usual_rails_template_path.delete_suffix("_mailer").pluralize
+
+      headers[:template_path] = [headers[:template_path],
+                                 usual_rails_template_path, custom_template_path]
+                                .flatten.compact
```

Comment:
Note that we keep the `template_path` header that the user set manually. E.g. you can always place mailers in a subfolder like `app/frontend/feedback/mails/` if you want to (e.g. this could be useful for very big topics). Just add `template_path: "feedback/mails/"` manually to the respective mail controller in this case.

Reply:
That's good.